offlinedocs/pages/[[...slug]].tsx

```diff
@@ -195,7 +196,7 @@ const getNavigation = (manifest: Manifest): Nav => {
 };
 
 const removeHtmlComments = (string: string) => {
-	return string.replace(/<!--[\s\S]*?-->/g, "");
+	return sanitizeHtml(string);
 };
```

Comment:
at this point, now we're just adding indirection to calling `sanitizeHtml`, and it looks like `removeHtmlComments` is only called in one place. can we just get rid of this function entirely and call `sanitizeHtml` directly there?